[2024-03-02T03:16:37.197+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_pipeline.merge_data scheduled__2024-03-02T01:01:00+00:00 [queued]>
[2024-03-02T03:16:37.382+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_pipeline.merge_data scheduled__2024-03-02T01:01:00+00:00 [queued]>
[2024-03-02T03:16:37.391+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-02T03:16:37.668+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-02 01:01:00+00:00
[2024-03-02T03:16:37.719+0000] {standard_task_runner.py:60} INFO - Started process 1803 to run task
[2024-03-02T03:16:37.739+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_pipeline', 'merge_data', 'scheduled__2024-03-02T01:01:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/ecom.py', '--cfg-path', '/tmp/tmpa19aj6dc']
[2024-03-02T03:16:37.763+0000] {standard_task_runner.py:88} INFO - Job 55: Subtask merge_data
[2024-03-02T03:16:37.932+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-02T03:16:38.196+0000] {task_command.py:423} INFO - Running <TaskInstance: ecommerce_pipeline.merge_data scheduled__2024-03-02T01:01:00+00:00 [running]> on host 70b88be0f8e4
[2024-03-02T03:16:43.628+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='merge_data' AIRFLOW_CTX_EXECUTION_DATE='2024-03-02T01:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-02T01:01:00+00:00'
[2024-03-02T03:16:43.634+0000] {logging_mixin.py:188} INFO - Error: Can only merge Series or DataFrame objects, a <class 'str'> was passed
[2024-03-02T03:16:43.635+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-02T03:16:43.663+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ecommerce_pipeline, task_id=merge_data, execution_date=20240302T010100, start_date=20240302T031637, end_date=20240302T031643
[2024-03-02T03:16:43.776+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-02T03:16:43.844+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
